pyproject.toml

```
@@ -69,6 +69,7 @@ jpeg2000 = [
   "lxml>=4.9.0,!=5.0.0",
 ]
 map = [
+  "contourpy>=1.2.0",
```

```suggestion
  "contourpy>=1.0.0",
```
Do we need to pin a specific version for MPL?

---

As mentioned [here](https://github.com/sunpy/sunpy/pull/5923#discussion_r819417509) we can go with `>=v1.0.1`

---

Sounds good